packager/mpd/base/period.h

```diff
@@ -130,7 +131,8 @@ class Period {
                const MediaInfo& media_info);
     // Check if the two adaptation sets are switchable.
     bool Switchable(const AdaptationSet& adaptation_set_a,
-                    const AdaptationSet& adaptation_set_b);
+                    const AdaptationSet& adaptation_set_b,
+                    const bool allow_codec_switching);
```

Comment:
remove const here. We don't prefix primitive types with const in this code base.